
public class ExceptionTest {

	public static void main(String[] args) {

		String str = null;

		try { // 예외가 발생할지도 모르는 코드

			str.toString();
			System.out.println("되나?");
			
		} catch (NullPointerException e) {
			System.out.println("객체생성하고 쓰라");
		}
	}
}
